helm/hocs-casework-search-indexer/values.yaml

```diff
@@ -1,13 +1,18 @@
 ---
-hocs-backend-service:
-  nameOverride: hocs-casework-search-indexer
 
-  app:
-    image:
-      repository: quay.io/ukhomeofficedigital/hocs-casework-search-indexer
-    resources:
-      limits:
-        cpu: 2000m
-        memory: 2048Mi
-      requests:
-        memory: 2048Mi
+app:
+  image:
+    repository: quay.io/ukhomeofficedigital/hocs-casework-search-indexer
+  resources:
```

Comment:
As discussed not super important but I'd investigate matching request and limit to get into the guaranteed QoS 
https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/